cmd/clusterctl/client/cluster/cert_manager.go

```diff
@@ -72,15 +88,8 @@ func newCertMangerClient(configClient config.Client, proxy Proxy, pollImmediateW
 }
 
 // Images return the list of images required for installing the cert-manager.
+// TODO: should this method return an empty slice if cert-manager is already installed?
```

Comment:
This method is used when we run `clusterctl init --list-images`. So I don't believe that it should ever return an empty slice.